ruby/lib/gherkin3/parser.rb

```diff
@@ -63,9 +63,12 @@ def initialize(ast_builder=AstBuilder.new)
       @ast_builder = ast_builder
     end
 
-    def parse(token_scanner, token_matcher=TokenMatcher.new)
+    def parse(input, token_matcher=TokenMatcher.new)
```

Comment:
It does! https://github.com/dg-ratiodata/gherkin3/commit/d9085a51cc39f516795f2ef2b0b5f6bdcb415a81#diff-a5a70ea60d3411ec934a6b3412ff8af6R31